threedi_modelchecker/exporters.py

```diff
@@ -77,6 +87,8 @@ def generate_csv_table(checks) -> str:
 
     writer.writeheader()
 
+    checks = order_checks(checks)
+
     for check in checks:
```

Comment:
More a style suggestion. I'd not use an assignment in this case because it just adds unnecessary clutter.

```suggestion
    for check in order_checks(checks):
```